### ISSUE TITLE
Drivers support for REEF models please :)

### ISSUE THREAD

**Issue by Universal-Igloo** [MilkyDeveloper/cb-linux#1]
Hi, thank you for your work! Would love to have support for the apollolake REEF models, mine is the Asus Flip C213.

Here are the outputs of the 2 commands requested -

-----------

lsmod:

Module                  Size  Used by
vhost_vsock            20480  5
vmw_vsock_virtio_transport_common    20480  1 vhost_vsock
vsock                  32768  7 vmw_vsock_virtio_transport_common,vhost_vsock
vhost                  24576  1 vhost_vsock
veth                   16384  0
rfcomm                 40960  2
cmac                   16384  3
xt_cgroup              16384  2
algif_hash             16384  1
algif_skcipher         20480  1
af_alg                 16384  6 algif_hash,algif_skcipher
uinput                 20480  0
ipt_MASQUERADE         16384  1
nf_nat_masquerade_ipv4    16384  1 ipt_MASQUERADE
xt_mark                16384  78
iio_trig_hrtimer       16384  2
industrialio_sw_trigger    16384  1 iio_trig_hrtimer
industrialio_configfs    16384  2 industrialio_sw_trigger
snd_soc_sst_bxt_da7219_max98357a    24576  4
snd_soc_hdac_hdmi      24576  1 snd_soc_sst_bxt_da7219_max98357a
snd_soc_dmic           16384  1
snd_soc_skl_ssp_clk    16384  0
snd_soc_skl            65536  3 snd_soc_skl_ssp_clk
snd_soc_skl_ipc        36864  1 snd_soc_skl
snd_soc_sst_dsp        32768  1 snd_soc_skl_ipc
snd_soc_sst_ipc        16384  1 snd_soc_skl_ipc
snd_soc_sst_match      16384  1 snd_soc_skl
snd_hda_ext_core       20480  2 snd_soc_hdac_hdmi,snd_soc_skl
snd_hda_core           53248  3 snd_hda_ext_core,snd_soc_hdac_hdmi,snd_soc_skl
acpi_als               16384  1
snd_soc_max98357a      16384  1
snd_soc_da7219         53248  2 snd_soc_sst_bxt_da7219_max98357a
ip6table_nat           16384  1
nf_nat_ipv6            16384  1 ip6table_nat
fuse                   90112  0
ccm                    20480  3
ip6table_filter        16384  1
iio_trig_sysfs         16384  0
cros_ec_sensors        16384  2
cros_ec_sensors_core    16384  1 cros_ec_sensors
industrialio_triggered_buffer    16384  2 cros_ec_sensors_core,acpi_als
kfifo_buf              16384  2 industrialio_triggered_buffer,cros_ec_sensors_core
industrialio           49152  9 iio_trig_hrtimer,industrialio_triggered_buffer,iio_trig_sysfs,cros_ec_sensors_core,acpi_als,kfifo_buf,cros_ec_sensors
cros_ec_sensorhub      20480  1 cros_ec_sensors_core
zram                   24576  1
iwlmvm                409600  0
iwlwifi               344064  1 iwlmvm
iwl7000_mac80211      655360  2 iwlwifi,iwlmvm
cfg80211              479232  3 iwlwifi,iwlmvm,iwl7000_mac80211
btusb                  40960  0
btrtl                  16384  1 btusb
btintel                24576  1 btusb
btbcm                  16384  1 btusb
bluetooth             413696  32 btbcm,btrtl,btusb,rfcomm,btintel
uvcvideo               81920  0
videobuf2_vmalloc      16384  1 uvcvideo
videobuf2_v4l2         24576  1 uvcvideo
videobuf2_core         28672  2 uvcvideo,videobuf2_v4l2
videobuf2_memops       16384  1 videobuf2_vmalloc
joydev                 20480  0

----------

find /usr/share/alsa:

/usr/share/alsa
/usr/share/alsa/cards
/usr/share/alsa/cards/aliases.conf
/usr/share/alsa/cards/ICE1724.conf
/usr/share/alsa/cards/NFORCE.conf
/usr/share/alsa/cards/ATIIXP.conf
/usr/share/alsa/cards/Echo_Echo3G.conf
/usr/share/alsa/cards/VX222.conf
/usr/share/alsa/cards/vc4-hdmi.conf
/usr/share/alsa/cards/ATIIXP-MODEM.conf
/usr/share/alsa/cards/CMI8738-MC8.conf
/usr/share/alsa/cards/AACI.conf
/usr/share/alsa/cards/Maestro3.conf
/usr/share/alsa/cards/HdmiLpeAudio.conf
/usr/share/alsa/cards/VIA686A.conf
/usr/share/alsa/cards/ICH-MODEM.conf
/usr/share/alsa/cards/Aureon71.conf
/usr/share/alsa/cards/HDA-Intel.conf
/usr/share/alsa/cards/AU8820.conf
/usr/share/alsa/cards/CMI8338.conf
/usr/share/alsa/cards/FWSpeakers.conf
/usr/share/alsa/cards/Audigy2.conf
/usr/share/alsa/cards/RME9636.conf
/usr/share/alsa/cards/VXPocket.conf
/usr/share/alsa/cards/EMU10K1X.conf
/usr/share/alsa/cards/ENS1371.conf
/usr/share/alsa/cards/CMI8738-MC6.conf
/usr/share/alsa/cards/VXPocket440.conf
/usr/share/alsa/cards/TRID4DWAVENX.conf
/usr/share/alsa/cards/CMI8338-SWIEC.conf
/usr/share/alsa/cards/Audigy.conf
/usr/share/alsa/cards/SI7018.conf
/usr/share/alsa/cards/ES1968.conf
/usr/share/alsa/cards/ENS1370.conf
/usr/share/alsa/cards/AU8810.conf
/usr/share/alsa/cards/VIA8233.conf
/usr/share/alsa/cards/CS46xx.conf
/usr/share/alsa/cards/pistachio-card.conf
/usr/share/alsa/cards/PS3.conf
/usr/share/alsa/cards/EMU10K1.conf
/usr/share/alsa/cards/CMI8788.conf
/usr/share/alsa/cards/PC-Speaker.conf
/usr/share/alsa/cards/CA0106.conf
/usr/share/alsa/cards/ATIIXP-SPDMA.conf
/usr/share/alsa/cards/RME9652.conf
/usr/share/alsa/cards/PMac.conf
/usr/share/alsa/cards/AU8830.conf
/usr/share/alsa/cards/VIA8233A.conf
/usr/share/alsa/cards/USB-Audio.conf
/usr/share/alsa/cards/GUS.conf
/usr/share/alsa/cards/ICH.conf
/usr/share/alsa/cards/PMacToonie.conf
/usr/share/alsa/cards/FM801.conf
/usr/share/alsa/cards/Loopback.conf
/usr/share/alsa/cards/ICH4.conf
/usr/share/alsa/cards/ICE1712.conf
/usr/share/alsa/cards/SB-XFi.conf
/usr/share/alsa/cards/VIA8237.conf
/usr/share/alsa/cards/YMF744.conf
/usr/share/alsa/cards/FireWave.conf
/usr/share/alsa/cards/Aureon51.conf
/usr/share/alsa/speaker-test
/usr/share/alsa/speaker-test/sample_map.csv
/usr/share/alsa/ucm
/usr/share/alsa/ucm/Poly Blackwire 8225 Series
/usr/share/alsa/ucm/Poly Blackwire 8225 Series/Poly Blackwire 8225 Series.conf
/usr/share/alsa/ucm/Poly Blackwire 8225 Series/HiFi.conf
/usr/share/alsa/ucm/Plantronics DA70
/usr/share/alsa/ucm/Plantronics DA70/Plantronics DA70.conf
/usr/share/alsa/ucm/Plantronics DA70/HiFi.conf
/usr/share/alsa/ucm/Jabra SPEAK 810
/usr/share/alsa/ucm/Jabra SPEAK 810/HiFi.conf
/usr/share/alsa/ucm/Jabra SPEAK 810/Jabra SPEAK 810.conf
/usr/share/alsa/ucm/Dell AC511 USB SoundBar
/usr/share/alsa/ucm/Dell AC511 USB SoundBar/Dell AC511 USB SoundBar.conf
/usr/share/alsa/ucm/Dell AC511 USB SoundBar/HiFi.conf
/usr/share/alsa/ucm/Pixel USB-C earbuds
/usr/share/alsa/ucm/Pixel USB-C earbuds/HiFi.conf
/usr/share/alsa/ucm/Pixel USB-C earbuds/Pixel USB-C earbuds.conf
/usr/share/alsa/ucm/.keep_media-sound_alsa-utils-0.9
/usr/share/alsa/ucm/Scarlett Solo USB
/usr/share/alsa/ucm/Scarlett Solo USB/HiFi.conf
/usr/share/alsa/ucm/Scarlett Solo USB/Scarlett Solo USB.conf
/usr/share/alsa/ucm/PCP-USB
/usr/share/alsa/ucm/PCP-USB/HiFi.conf
/usr/share/alsa/ucm/PCP-USB/PCP-USB.conf
/usr/share/alsa/ucm/Chat 150 C
/usr/share/alsa/ucm/Chat 150 C/HiFi.conf
/usr/share/alsa/ucm/Chat 150 C/Chat 150 C.conf
/usr/share/alsa/ucm/ICUSBAUDIO7D
/usr/share/alsa/ucm/ICUSBAUDIO7D/HiFi.conf
/usr/share/alsa/ucm/ICUSBAUDIO7D/ICUSBAUDIO7D.conf
/usr/share/alsa/ucm/Scarlett 2i2 USB
/usr/share/alsa/ucm/Scarlett 2i2 USB/HiFi.conf
/usr/share/alsa/ucm/Scarlett 2i2 USB/Scarlett 2i2 USB.conf
/usr/share/alsa/ucm/Hangouts Meet speakermic
/usr/share/alsa/ucm/Hangouts Meet speakermic/HiFi.conf
/usr/share/alsa/ucm/Hangouts Meet speakermic/Hangouts Meet speakermic.conf
/usr/share/alsa/ucm/sof-hda-dsp
/usr/share/alsa/ucm/sof-hda-dsp/HiFi.conf
/usr/share/alsa/ucm/sof-hda-dsp/sof-hda-dsp.conf
/usr/share/alsa/ucm/Plantronics EncorePro 525 USB
/usr/share/alsa/ucm/Plantronics EncorePro 525 USB/HiFi.conf
/usr/share/alsa/ucm/Plantronics EncorePro 525 USB/Plantronics EncorePro 525 USB.conf
/usr/share/alsa/ucm/Mi Dual Driver Earphones Type-C
/usr/share/alsa/ucm/Mi Dual Driver Earphones Type-C/Mi Dual Driver Earphones Type-C.conf
/usr/share/alsa/ucm/Mi Dual Driver Earphones Type-C/HiFi.conf
/usr/share/alsa/ucm/bxtda7219max.reef.ELECTRO
/usr/share/alsa/ucm/bxtda7219max.reef.ELECTRO/HiFi.conf
/usr/share/alsa/ucm/bxtda7219max.reef.ELECTRO/bxtda7219max.reef.ELECTRO.conf
/usr/share/alsa/ucm/Sennheiser USB headset
/usr/share/alsa/ucm/Sennheiser USB headset/HiFi.conf
/usr/share/alsa/ucm/Sennheiser USB headset/Sennheiser USB headset.conf
/usr/share/alsa/ucm/DELL PROFESSIONAL SOUND BAR AE5
/usr/share/alsa/ucm/DELL PROFESSIONAL SOUND BAR AE5/HiFi.conf
/usr/share/alsa/ucm/DELL PROFESSIONAL SOUND BAR AE5/DELL PROFESSIONAL SOUND BAR AE5.conf
/usr/share/alsa/ucm/Dell-WD15-Dock
/usr/share/alsa/ucm/Dell-WD15-Dock/Dell-WD15-Dock.conf
/usr/share/alsa/ucm/Dell-WD15-Dock/HiFi.conf
/usr/share/alsa/ucm/Scarlett 2i4 USB
/usr/share/alsa/ucm/Scarlett 2i4 USB/HiFi.conf
/usr/share/alsa/ucm/Scarlett 2i4 USB/Scarlett 2i4 USB.conf
/usr/share/alsa/ucm/Logitech BRIO
/usr/share/alsa/ucm/Logitech BRIO/HiFi.conf
/usr/share/alsa/ucm/Logitech BRIO/Logitech BRIO.conf
/usr/share/alsa/ucm/Loopback
/usr/share/alsa/ucm/Loopback/HiFi.conf
/usr/share/alsa/ucm/Loopback/Loopback.conf
/usr/share/alsa/ucm/WD19 Dock
/usr/share/alsa/ucm/HUAWEI USB-C HEADSET
/usr/share/alsa/ucm/HUAWEI USB-C HEADSET/HiFi.conf
/usr/share/alsa/ucm/HUAWEI USB-C HEADSET/HUAWEI USB-C HEADSET.conf
/usr/share/alsa/ucm/WD15 Dock
/usr/share/alsa/ucm/Logitech Webcam C930e
/usr/share/alsa/ucm/Logitech Webcam C930e/HiFi.conf
/usr/share/alsa/ucm/Logitech Webcam C930e/Logitech Webcam C930e.conf
/usr/share/alsa/ucm/C505 HD Webcam
/usr/share/alsa/ucm/C505 HD Webcam/HiFi.conf
/usr/share/alsa/ucm/C505 HD Webcam/C505 HD Webcam.conf
/usr/share/alsa/ucm/USB 2.0 Camera
/usr/share/alsa/ucm/USB 2.0 Camera/HiFi.conf
/usr/share/alsa/ucm/USB 2.0 Camera/USB 2.0 Camera
/usr/share/alsa/ucm/bxtda7219max.reef.BASKING
/usr/share/alsa/ucm/bxtda7219max.reef.BASKING/bxtda7219max.reef.BASKING.conf
/usr/share/alsa/ucm/bxtda7219max.reef.BASKING/HiFi.conf
/usr/share/alsa/pcm
/usr/share/alsa/pcm/surround50.conf
/usr/share/alsa/pcm/dmix.conf
/usr/share/alsa/pcm/surround51.conf
/usr/share/alsa/pcm/dpl.conf
/usr/share/alsa/pcm/center_lfe.conf
/usr/share/alsa/pcm/hdmi.conf
/usr/share/alsa/pcm/surround40.conf
/usr/share/alsa/pcm/rear.conf
/usr/share/alsa/pcm/iec958.conf
/usr/share/alsa/pcm/side.conf
/usr/share/alsa/pcm/modem.conf
/usr/share/alsa/pcm/surround41.conf
/usr/share/alsa/pcm/default.conf
/usr/share/alsa/pcm/surround21.conf
/usr/share/alsa/pcm/dsnoop.conf
/usr/share/alsa/pcm/front.conf
/usr/share/alsa/pcm/surround71.conf
/usr/share/alsa/init
/usr/share/alsa/init/00main
/usr/share/alsa/init/test
/usr/share/alsa/init/info
/usr/share/alsa/init/default
/usr/share/alsa/init/hda
/usr/share/alsa/init/help
/usr/share/alsa/init/ca0106
/usr/share/alsa/alsa.conf

**Comment by MilkyDeveloper** [comment]
Fixed :D! Check the docs.

**Comment by MilkyDeveloper** [comment]
I was just wondering: from a user's perspective, is it clear that all chromebook drivers are supported except certain audio drivers? Is there anything I could do to make it more clear?

**Comment by Universal-Igloo** [comment]
Thank you! I am so excited to try this out once I can setup a linux machine to build it from!

> I was just wondering: from a user's perspective, is it clear that all chromebook drivers are supported except certain audio drivers? Is there anything I could do to make it more clear?
It's pretty clear now, you could consider making the words *audio support* BOLD in this first line - "However, this project is being developed on Nami, so the following models will have audio support:"

**Comment by jeffzhang128** [comment]
I've run the command to setup audio for my Acer Spin 11 R751T (reef), but the command fails at `mount: /mnt: special device /dev/mmcblk0p3 does not exist.`which I believe is because the internal eMMC storage is always located at /dev/mmcblk1 instead of /dev/mmcblk0 on this model. Because of this, audio is not setup properly on my Chromebook.

**Comment by jeffzhang128** [comment]
Another issue I've found on my Acer Spin 11 is that unfortunately, I still haven't managed to get audio working on KDE. I'm not sure whether this would be a problem with the desktop environment or something else, but I have noticed that `plasma-pa`, `libcanberra-pulse`, and `pulseaudio-utils` all get removed the `sudo apt purge pulseaudio*` line in the script to install them without `pulseaudio` as it was explicitly removed by the script. By doing that, I've managed to get back the audio control button on KDE, but it shows that there aren't any audio input or output devices. While I still think that it is pretty unbelievable for us to finally have audio working in linux on Apollo Lake devices, I really don't know what I'm doing so any idea of what could be going wrong is greatly appreciated.
